src/theme/dojo/loading-indicator.m.css

```diff
@@ -1,4 +1,7 @@
 .root {
```

Comment:
Should the dojo theme have the size classes with the appropriate CSS Vars?

Reply:
I've added the `.small` `.medium` and `.large` classes to the dojo and material theme, however should the sizes differ from the default sizing?

Reply:
As long as they use CSS variables and they are configured in the variants for dojo/material default and dark then they can be the same as the default sizes - that will allow users to create a custom variant and change them